demo/app.js

```diff
@@ -246,7 +248,8 @@ const Demo = React.createClass({
             name: 'Data Point 3',
             value: 200
           }
-        ]
+        ],
+        gaugeData: []
```

Comment:
If guageData isn't changing from the initial state then I would say we don't need to include this here on `componentDidMount`